nss-day-cohort-46/rare-client-devious-dragons
# Bs filterposts

# Description
Added category sort dropdown

Fixes # (issue)

## Type of change


# Testing Instructions

```
git fetch --all
git checkout bs-filterposts

go to main post list
select a user/category from dropdown
list should render only posts from that user/category
select All users/categories should reset to all posts by date list

```

# Checklist:
- [x] Proper filtering
- [x] Other dropdown resets when selecting
